pr-data.csv

```diff
@@ -3270,8 +3271,8 @@ https://github.com/apache/openwebbeans,01245cf8ef0baaa9cd31342b0cdb075d314b6ef9,
 https://github.com/apache/openwebbeans,01245cf8ef0baaa9cd31342b0cdb075d314b6ef9,webbeans-impl,org.apache.webbeans.test.specalization.observer.pub.PublicObserverTest.testObserverMethodsInParentOfAlternativeAndSpecializedBeans,ID,Opened,https://github.com/apache/openwebbeans/pull/100,
 https://github.com/apache/openwebbeans,01245cf8ef0baaa9cd31342b0cdb075d314b6ef9,webbeans-impl,org.apache.webbeans.test.specalization.passivation.SpecializesPassivationTest.testSpecializePassivatingBean,ID,Opened,https://github.com/apache/openwebbeans/pull/100,
 https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-impl,org.apache.webbeans.test.unittests.specializes.logger.LoggerSpecializationTest.testSpecializedVersion,ID,Opened,https://github.com/apache/openwebbeans/pull/100,https://github.com/TestingResearchIllinois/idoft/issues/561
-https://github.com/apache/openwebbeans,424af4bb7285b806d3e3efe5020cd3e440e0a4ff,webbeans-jetty9,org.apache.webbeans.web.jetty9.test.JettyNormalScopeProxyFactoryTest.checkDeserialisation,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/561
```

Comment:
Do not delete the old info but add new with a semicolon (`;`) separating the links.